charts/agent-stack-k8s/values.schema.json

```
@@ -36,6 +36,25 @@
         }
       ]
     },
+    "labels": {
+      "type": "object",
+      "default": {},
+      "title": "The labels Schema",
+      "additionalProperties": {
+        "type": "object",
+        "default": {}
+      },
+      "example": [
+        { 
+          "user": {
+            "user": "name"
+          },
+          "project": {
+            "project": "project"
+          }
+        }
+      ]
```

Are these valid labels? My understanding is that labels are key-value pairs where both the key and value are strings. But here it seems you're asking for the values to be objects.

I tried to deploy this from your branch with
```
labels:
  user:
    user: "name"
  project:
    project: "project"
```
which is the example given and passes the JSON schema, but Kubernetes does not accept it with the error message:
```
json: cannot unmarshal object into Go struct field ObjectMeta.spec.template.metadata.labels of type string
```

I think it needs to be something like:
```suggestion
      "additionalProperties": {
        "type": "string",
        "default": ""
      },
      "example": [
        { 
          "user": "name",
          "project": "project"
        }
      ]
```